### ISSUE TITLE
Installation instructions are missing a wheel?

### ISSUE THREAD

**Issue by GreyGnome** [dell/omsdk#8]
# QwikContext
The Installation instructions seem to be missing a wheel that I have.

# Details
After sh build.sh 1.2 345, there are two wheels created:

omdrivers-1.2.345_-py2.py3-none-any.whl   omsdk-1.2.345_-py2.py3-none-any.whl

# Repro Steps

- Grab the Zip file from the GitHub homepage
- Unzip it
- cd omsdk-master
- sh build.sh 1.2 345
- cd dist
- You'll see 2 files there.
- I performed:  pip install --user omdrivers-1.2.345_-py2.py3-none-any.whl omsdk-1.2.345_-py2.py3-none-any.whl

# Impact
No problem for me.

# NeedBy
Anytime. Is it even a problem? But then, if not, the additional file is spurious and confusing.

# Severity
Low.

**Comment by anupamaloke** [comment]
@GreyGnome, you are right in that there are two wheel packages created when you build omsdk. By default, ```pip install omsdk-<version>-py2.py3.none-any.whl``` will also install the ```omdriver``` wheel. So there is no need to install ```omdriver``` wheel package separately.
